daycare/models.py

```diff
@@ -39,3 +47,24 @@ def create(email, username, password, name, description, price_per_day, capacity
         new_daycare.save()
 
         return new_daycare
+
+
+class Image(models.Model):
+    name = models.CharField(max_length=255)
+    image = models.CharField(max_length=1000)
+    daycare_id = models.ForeignKey(DayCare, null=True, on_delete=models.SET_NULL)
```

Comment:
why not have a "uploader_id" and have this be a ForeignKey of User?

Reply:
Because we want that the class Image will refer only to daycares in order that they will be able to upload multiple images.
and for the dogOwner user, he has only one field with image_url

Reply:
I guess that makes sense, but a bit of a waste.